packages/admin-ui/src/dappnode_styles.scss

```diff
@@ -266,3 +266,8 @@ p.no-p-style:last-child {
     opacity: 0.7;
   }
 }
+
+.bottom-alert {
```

Comment:
styles applied in the dappnode_styles.scss should be generic and used in multiple places in the dappmanager. Also, to gain space at the top and bottom you could simply use the `<br />` HTML element

Reply:
I tried to use the <br/> tag, but it did not work with the alert component. I would rather create a CSS file for this component, but I am not sure where to create it or if it is OK to do it like that.

Reply:
Fixed on https://github.com/dappnode/DNP_DAPPMANAGER/pull/1190/commits/1342757c0019451e8ef5f2e5a2fafc46d3f3deae